app/views/layouts/application.haml

```diff
@@ -0,0 +1,45 @@
+!!!
+%html
+  %head
+    = stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true
+    = javascript_include_tag 'application', 'data-turbolinks-track' => true
+    = csrf_meta_tags
+
+    %meta{:charset => "utf-8"}/
+    %meta{:content => "", :name => "description"}/
+    %meta{:content => "width=device-width, initial-scale=1", :name => "viewport"}/
+    %title Melange
+    / Place favicon.ico in the root directory
+    %link{:href => "public/favicon.ico", :rel => "shortcut icon"}/
+    / Mandatory Stylesheets
+    %link{:href => "css/base/bootstrap.css", :rel => "stylesheet"}/
+    %link{:href => "https://fonts.googleapis.com/css?family=Open+Sans:300,400,700,800", :rel => "stylesheet"}/
+    / Plugin Stylesheets
+    %link{:href => "css/plugins/owl-carousel/owl.carousel.min.css", :rel => "stylesheet"}/
```

Comment:
css files are loaded by the `stylesheet link tag` - you should not include them here